Publishing steps (#135)
mavenlink/brainstem-js: README.md

    yarn test-watch

## Publishing

1. Use an NPM account (with 2FA enabled) on the Mavenlink organization
2. Run `yarn install` to update dependencies
3. Modify the semantic version appropriately as a commit
4. Make a pull request with said changes

> 5. Run `yarn publish` to publish to NPM

Author: ```suggestion
5. If using a pre-release flag, run `yarn publish` to publish the alpha version to NPM
```

> 6. Request a review (and merge accordingly)

Author: ```suggestion
6. Request a review (and merge accordingly)
7. Run `yarn publish` on `master` to publish the latest release version to NPM
```

    yarn test-watch

## Publishing

1. Use an NPM account (with 2FA enabled) on the Mavenlink organization
2. Run `yarn install` to update dependencies
3. Modify the semantic version appropriately as a commit
4. Make a pull request with said changes

> 5. If using a pre-release flag, run `yarn publish` to publish the alpha version to NPM

Author: ```suggestion
5. If using a [pre-release tag](https://docs.npmjs.com/misc/semver#prerelease-tags), run `yarn publish` to publish the alpha version to NPM
```
